### ISSUE TITLE
[bug] `conan.tools.scm.Version` error

### ISSUE THREAD

**Issue by mpusz** [conan-io/conan#11105]
While trying to replace `conans.tools.Version` to `conan.tools.scm.Version` I get the following error

```
Configuration:
[settings]
arch=x86_64
arch_build=x86_64
build_type=Release
compiler=gcc
compiler.cppstd=20
compiler.libcxx=libstdc++11
compiler.version=11
os=Linux
os_build=Linux
[options]
mp-units:build_docs=False
[build_requires]
[env]
CC=/usr/bin/gcc-11
CONAN_RUN_TESTS=True
CXX=/usr/bin/g++-11
[conf]
tools.cmake.cmaketoolchain:generator=Ninja

ERROR: mp-units/0.8.0: Error in configure() method, line 116
        version = Version(self.settings.compiler.version)
        ConanException: 'settings.compiler.version.strip' doesn't exist for '11'
'settings.compiler.version' possible configurations are none
```

**Comment by memsharded** [comment]
Trying to submit a fix for it with https://github.com/conan-io/conan/pull/11119

In the meantime, you can avoid it by doing a ``Version(str(self.settings.compiler.version))``

**Comment by memsharded** [comment]
Hi @leha-bot 

In theory this has been fixed in Conan 2.0 (already in beta), but please feel free to open a new ticket if you still have any issues around this in 2.0. Thanks!
